### PR TITLE
Change screenshot scaling strategy

### PR DESCRIPTION
**Pull request by BenCheung0422**
Currently, bicubic interpolation is used for scaling screenshots into desired scales. ~~This adds a setting to control which interpolation is used for this.~~ (The interpolation type is entirely changed to nearest neighbor instead of adding a setting) In time complexity and file size, it is obvious that: bicubic > bilinear > nearest-neighbor. However, the quality of images differs from each other's. In usual, bicubic is commonly used for image and video processing, but the canvas of Minicraft is scaling with ratio magnification, which is nearest-neighbour. Using nearest-neighbour could make the screenshots closer to the look in-game.

### CODE REVIEW

**Comment by Makkkkus** [comment]
Don't make it a setting, just use nearest neighbour.

**Review by Makkkkus — CHANGES_REQUESTED** [review]
Still not fixed

**Comment by BenCheung0422** [comment]
Alright, I have changed.

**Review by ChristofferHolmesland — DISMISSED** [review]

File: src/client/resources/assets/localization/en-us.json [hunk 1]
```diff
@@ -282,7 +282,6 @@
   "minicraft.settings.mode.hardcore": "Hardcore",
   "minicraft.settings.mode.score": "Score",
   "minicraft.settings.scoretime": "Time (Score Mode)",
-  "minicraft.settings.screenshot_scale": "Screenshot Scale",
```
Comment (Makkkkus) [inline src/client/resources/assets/localization/en-us.json:285]: You removed a localisation entry?

**Review by Makkkkus — CHANGES_REQUESTED** [review]

File: src/client/resources/assets/localization/en-us.json (hunk 1 above)
Comment (BenCheung0422) [reply src/client/resources/assets/localization/en-us.json]: I have removed too many lines. I have added it back now.

**Review by Makkkkus — APPROVED** [review]

**Review by Litorom — APPROVED** [review]
Cool

**Pull request merged**